jools.html

```diff
@@ -0,0 +1,26 @@
+DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="../main.css" rel="stylesheet">
+    <title>Hello World</title>
+</head>
+<body>
+    <!-- add your name in any size of h tag: h1, h2, h3, h4, h5, h6  -->
+    <!-- <h1>Jools</h1> -->
+
+    <!-- If you want, add another image from online -->
+    <!-- <img src="https://images.pexels.com/photos/2023384/pexels-photo-2023384.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=2" class="your-image"> -->
```

Comment:
A couple things here: 
- This line is currently commented out, uncomment it out so it doesn't have <!-- or  -->
- You can change the image by including another link of your choice. 
To add an image from the internet:
                right click on the image and select Copy Image Address and paste it in between the quotes of the href attribute on the a tag